Package.resolved

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Do we need this file?

Reply:
Oh no. I removed it at https://github.com/amplitude/Amplitude-iOS/commit/c0cd582a855a52325294873e1f4ccab505f4a6b2